vue/sbc-common-components/src/models/notification.ts

```diff
@@ -0,0 +1,26 @@
+export enum AppType {
+  ALL = 'ALL',
+  Business = 'BUSINESS',
+  NameRequest = 'NAMEREQUEST',
+  PPR = 'PPR'
+}
+
+export enum NotificationLabel {
+  NEW = 'New Feature',
+  IMPROVEMENT = 'Improvement',
+  ANNOUNCEMENT = 'Announcement'
+}
+
+export interface Notification {
+  id: number;
+  date: string;
+  expiryDate?: string;
+  labels?: NotificationLabel[];
+  title: string;
+  description: string;
+  app: AppType;
+  priority: boolean;
+  read: boolean;
+}
+
+export type Notifications = Notification[];
```

Comment:
Sometimes lint requires a blank line at the end of a file.

Reply:
I added a blank line but my editor report linting error.

Reply:
Hahaha, OK, never mind!

Who knows the mysteries of Lint ?